examples/example-runtime/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 license = "GPL-3.0"
 
 [dependencies]
-ya-runtime-sdk = { version = "0.1", path = "../../ya-runtime-sdk", features = ["macros"] }
+ya-runtime-sdk = { version = "0.2", path = "../../ya-runtime-sdk", features = ["macros"] }
```

Comment:
I think that `version = "1.2.3"` should be changed to e.g. `version = "1.2.4"` if `ya-runtime-sdk` version is bumped.

Reply:
It probably isn't documented well enough, but the point is to present that the generated CLI prints out the version specified in `Cargo.toml`. It could be bumped though but I don't see much value in it.